docs/canvas.asciidoc

```diff
@@ -6,26 +6,22 @@
 [partintro]
 --
 
-Congratulations on finding the Canvas application in {kib}. You are in for a treat. 
-Canvas is a whole new way of making data look amazing. Canvas combines data with 
-colors, shapes, text, and your own imagination to bring dynamic, multi-page, 
-pixel-perfect, data displays to screens large and small. 
+Canvas is a data visualization and presentation tool that sits within Kibana. Backed by Elasticsearch, Canvas allows you to combine data with colors, images, text, and your imagination to bring dynamic, multi-page, pixel-perfect displays to screens large and small. If you are a little bit creative, a little bit technical, and a whole lot curious, then Canvas is for you. 
```

Comment:
`Canvas is a presentation tool — built into Kibana — that allows users to create pixel-perfect presentations and slide decks that pull live data directly from Elasticsearch. ` do you think this high level representation is good as a starting sentence ?

Reply:
I think we can consolidate these two into one paragraph. Let me see what I can do.